multimedia/M2VDownsizer/Portfile

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           makefile 1.0
+
+name                M2VDownsizer
+version             1.1
+revision            0
+
+categories          multimedia
+license             GPL-2
+maintainers         {hotmail.com:amtor @RobK88} \
+                    openmaintainer
+
+description         MPEG-2 video stream recompressesion
+
+long_description    ${name} is a tool that re-compresses MPEG-2 (or m2v) video \
+                    streams.  In other words, ${name} will perform a complete \
+                    decompression and recompression of MPEG-2 video streams for the \
+                    best possible video quality.\
+                    \n \
+                    \n${name} accepts the raw MPEG-2 (or m2v) video data (not VOB) \
+                    from the standard input and writes the recompressed video frames \
+                    to the standard output.\
+                    \n \
+                    \nBe forewarned!  Decompression and recompression could take \
+                    hours for large MPEG-2 video streams!  It is recommended that one \
+                    first tries transcoding MPEG-2 streams using M2VRequantiser since \
+                    transcoding is a lot faster and produces excellent results for most \
+                    MPEG-2 video streams.  ${name} is only intended to be used for \
+                    large MPEG-2 video streams when transcoding does not produce \
+                    acceptable results.
+
+homepage            https://web.archive.org/web/20120508050929if_/http://www.metakine.com/
+master_sites        ${homepage}files
+
+depends_lib         port:libmpeg2
+
+checksums           rmd160 d662bfe195b59f5815bfdccaf3f81dae46d4e0d8 \
+                    sha256 316a931810116d70a07cb57a80ae8483fa949bd575beee222a9a8e375c4d8590 \
+                    size 371928
+
+distname            ${name}
+
+dist_subdir         ${name}/${version}
+
+extract.suffix      .tgz
+
+post-extract {
+     copy ${filespath}/Makefile ${worksrcpath}
+     copy ${filespath}/${name}.1 ${worksrcpath}
+     file mkdir ${worksrcpath}/obj/utils
+     file mkdir ${worksrcpath}/obj/altivec
+     file mkdir ${worksrcpath}/obj/mpeg2enc
+     file mkdir ${worksrcpath}/obj/main
+}
+
+patchfiles          1001-Replace-MPProcessors-Patch.diff \
```

Comment:
I recommend using `patchfiles-append`, rather than `patchfiles`. Reason being, the latter replaces/removes any patches that have already been added.

For a case like this, where you only have one `patchfiles` statement, it's not a hugh deal. But it's still good practice, and can save time (and hair-pulling) when diagnosing portfile issues.

Ditto for things like `depends_xxx`: It's important to use `depends_build-append`, `depends_lib-append`, etc, to prevent wiping of any existing things added by macports base, and/or portgroups!

Reply:
> Ditto for things like `depends_xxx`: It's important to use `depends_build-append`, `depends_lib-append`, etc, to prevent wiping of any existing things added by macports base, and/or portgroups!

p.s. I realize you don't have any of these statements presently, but just an FYI for future work!

Reply:
Thanks Christopher for the tips!  I am leaning something new every port I work on.

I tested the portfile using "patchfiles-append" in my local repo and it worked without any issues.
Do you want me to change `patchfiles` to `patchfiles-append` for this port and push the change?

Reply:
> Thanks Christopher for the tips! I am leaning something new every port I work on.
> 
> I tested the portfile using "patchfiles-append" in my local repo and it worked without any issues. Do you want me to change `patchfiles` to `patchfiles-append` for this port and push the change?

Sure, go ahead and push!

Reply:
The push is done!